manifests/init.pp

```diff
@@ -24,6 +24,14 @@
 # node default {
 #   include nginx
 # }
+#
+# @param nginx_version
+#   The version of nginx installed (or being installed).
+#   Unfortunately, different versions of nginx may need configuring
+#   differently.  The default is derived from the version of nginx
+#   already installed.  If the fact is unavailable, it defaults to '1.6.0'.
```

Comment:
`1.6.0`? Did you mean `1.16.0`?

Reply:
No, that wasn't a typo.  See commit message or https://github.com/voxpupuli/puppet-nginx/pull/1330#issue-276960603  I wanted to pick a number < 1.15.0 and beyond that it didn't really matter what.  Debian 8 is a supported OS that ships with 1.6.x, so that seemed as good as any other to go with.

Maybe in a separate PR you might want to implement some of
> Future enhancements could make the non-fact default be OS specific.